spec/lib/publisher_spec.rb

```diff
@@ -6,9 +6,13 @@
   include GdsApi::TestHelpers::PublishingApi
   include GdsApi::TestHelpers::Rummager
 
+  let(:indexer) { double(:indexer) }
+
   before do
     stub_default_publishing_api_put
     stub_any_rummager_post
+    allow(SearchIndexer).to receive(:new).with(policy).and_return(indexer)
```

Comment:
If `SearchIndexer` is being made a double, then is the line above that stubs rummager redundant?